### ISSUE TITLE
Form params with multiple values

### ISSUE THREAD

**Issue by mperham** [sdogruyol/kemal#156]
I have a form which submits multiple values for a key:

```
key=a&key=b&key=c
```

I'd expect to be able to do `env.request.params.body["key"]` in Kemal and get back `["a", "b", "c"]` but the ParamParser seems to only support single-valued parameters.  How can I handle a form parameter with multiple values?

**Comment by mperham** [comment]
This is the workaround I found:

```
  bdy = HTTP::Params.parse(env.request.body.not_nil!)
  bdy.fetch_all("key").each do |key|
 ```

**Comment by sdogruyol** [comment]
For now `ParamParser` treats each key as a single value. 

What's your use case for this?

**Comment by mperham** [comment]
This is how you typically implement a list of checkboxes.

```erb
<% items.each do |item| %>
  <input type="checkbox" name="key[]" value="<%= value_for(item) %>" />
<% end %>
```

**Comment by sdogruyol** [comment]
Closed via #161
